PGM/src/main/java/tc/oc/pgm/polls/commands/PollSubCommands.java

```diff
@@ -0,0 +1,183 @@
+    @Inject
+    PollSubCommands(RestartManager restartManager,
+                 MutationQueue mutationQueue,
+                 PollManager pollManager,
+                 PollCustom.Factory pollCustomFactory,
+                 PollNextMap.Factory pollMapFactory,
+                 PollMutation.Factory pollMutationFactory,
+                 PollKick.Factory pollKickFactory,
+                 PollBlacklist pollBlacklist) {
+        this.restartManager = restartManager;
+        this.mutationQueue = mutationQueue;
+        this.pollManager = pollManager;
+        this.pollCustomFactory = pollCustomFactory;
+        this.pollMapFactory = pollMapFactory;
+        this.pollMutationFactory = pollMutationFactory;
+        this.pollKickFactory = pollKickFactory;
+        this.pollBlacklist = pollBlacklist;
+    }
+
+
+
+    @Command(
+        aliases = {"kick"},
+        desc = "Start a poll to kick another player.",
+        usage = "[player]",
+        min = 1,
+        max = 1
+    )
+    @CommandPermissions("poll.kick")
+    public void pollKick(CommandContext args, CommandSender sender) throws CommandException {
+        Player initiator = tc.oc.commons.bukkit.commands.CommandUtils.senderToPlayer(sender);
+        Player player = tc.oc.commons.bukkit.commands.CommandUtils.findOnlinePlayer(args, sender, 0);
+
+        if(player.hasPermission("pgm.poll.kick.exempt") && !initiator.hasPermission("pgm.poll.kick.override")) {
+            throw new TranslatableCommandException("poll.kick.exempt");
+        } else {
+            pollManager.startPoll(pollKickFactory.create(initiator.getName(), player));
+        }
+    }
+
+    @Command(
+        aliases = {"next"},
+        desc = "Start a poll to change the next map.",
+        usage = "[map name]",
+        min = 1,
+        max = -1
+    )
+    @CommandPermissions("poll.next")
+    public List<String> pollNext(CommandContext args, CommandSender sender) throws CommandException {
+        final String mapName = args.argsLength() > 0 ? args.getJoinedStrings(0) : "";
+        if(args.getSuggestionContext() != null) {
+            return CommandUtils.completeMapName(mapName);
+        }
+
+        if (!Config.Poll.enabled()) {
+            throw new TranslatableCommandException("poll.disabled");
+        }
+
+        if (restartManager.isRestartRequested()) {
+            throw new TranslatableCommandException("poll.map.restarting");
+        }
+        if (PGM.getMatchManager().hasMapSet()) {
+            throw new TranslatableCommandException("poll.map.alreadyset");
+        }
+        if (sender instanceof Player) {
+            Player player = (Player) sender;
+            if (TokenUtil.getUser(player).maptokens() < 1) {
+                throw new TranslatableCommandException("tokens.map.fail");
+            }
+        }
+
+        PGMMap nextMap = CommandUtils.getMap(args.getJoinedStrings(0), sender);
+
+        if (pollBlacklist.isBlacklisted(nextMap) && !sender.hasPermission("poll.next.override")) {
```

Comment:
Conventions are to not have a space between keywords and statements (ie. `if(...)`)